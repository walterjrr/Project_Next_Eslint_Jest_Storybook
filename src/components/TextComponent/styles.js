/* eslint-disable no-unused-vars */
import styled, { css } from 'styled-components';

export const Container = styled.p`
  ${({ theme }) => css`
    font-size: ${theme.font.sizes.medium};
  `};
`;
