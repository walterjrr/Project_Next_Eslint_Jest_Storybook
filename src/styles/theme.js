export const theme = {
  colors: {
    primaryColor: '#0A1128',
    secondaryColor: '#dc143c',
    white: '#ffffff',
  },
  fonts: {
    family: {
      default: "'Opens Sans', sans-serif",
      secondary: "'Montserrat', sans-serif",
    },
  },
  spacings: {},
};
