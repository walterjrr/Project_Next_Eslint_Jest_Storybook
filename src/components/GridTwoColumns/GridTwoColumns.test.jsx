import { renderTheme } from '../../styles/render-theme';
import { GridTwoColumns } from '.';
import { screen } from '@testing-library/react';

describe('<GridTwoColumns />', () => {
  it('should render', () => {
    renderTheme(<GridTwoColumns>Children</GridTwoColumns>);
    expect(screen.getByRole('heading')).toBeInTheDocument();
  });
});
