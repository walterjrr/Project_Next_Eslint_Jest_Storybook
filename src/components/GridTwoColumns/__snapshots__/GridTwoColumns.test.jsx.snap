// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GridTwoColumns /> should render 1`] = `
.c5 {
  color: #0A1128;
  font-size: 4.0rem;
  text-transform: uppercase;
}

.c2 {
  display: grid;
  grid-template-columns: 1fr 2fr;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  gap: 3.2rem;
}

.c2 .c4 {
  margin-bottom: 4.0rem;
}

.c1 {
  max-width: 120rem;
  margin: 0 auto;
  padding: 3.2rem;
}

.c0 {
  background: #FFFFFF;
  color: #0A1128;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c6 {
  font-size: 2.4rem;
}

@media (max-width:768px) {
  .c2 {
    grid-template-columns: 1fr;
  }
}

@media (max-width:768px) {
  .c3 {
    margin-bottom: 3.2rem;
  }
}

<div>
  <div
    class="c0"
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <div
          class="c3"
        >
          <h1
            class="c4 c5"
          >
            <h1>
              Grid Two Columns
            </h1>
          </h1>
          <p
            class="c6"
          >
            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Nostrum consequuntur perspiciatis nobis vitae optio deserunt mollitia ut inventore minima, illum facilis quia repellendus officiis nisi at odit dolor rem temporibus!
             
          </p>
        </div>
        <div
          class=""
        >
          <img
            alt="Grid Two Columns"
            class=""
            src="assets/images/logo.svg"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
